DOCKER.md

```diff
@@ -28,7 +28,7 @@ The docker image contains:
 Docker is very flexible so you can use that information however you choose. This guide isn't meant to be prescriptive.
 
 
-### Example: Running loopd
```

Comment:
🤔

Reply:
It was a weird non-breaking space that probably got in there through copy/paste. My IDE showed it so I removed it.